walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,83 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public final class FindMeetingQuery {
+  /**
+   * Finds all the time ranges in which the requested meeting can take place.
```

Comment:
I think this Javadoc should be more descriptive. 
In your API, you explicitly make the following decision:
If an optional attendee(s) was requested, first try to find something that works **with** the optional attendee. Otherwise, provide a range that doesn't necessarily work. 
This is an important part of the API, as you're not expected to get a bunch of time ranges that works either with or without the optional, but rather some order between them.

Reply:
Thanks, I updated the documentation. Done.